api/test/pact/constants/work-allocation/tasks.spec.ts

```diff
@@ -85,7 +85,18 @@ const AL_REDDY_DUNNE = {
   location: 'Dodgy Location',
   taskName: 'Assess errors',
   dueDate: new Date(),
-  actions: [ ACTIONS.REASSIGN ]
+  actions: [ ACTIONS.CLAIM, ACTIONS.REASSIGN ]
+};
+
+const NAT_ALLOWED_JR = {
```

Comment:
I'd probably call this NAT_AUTHORISED but I like the idea of a Jr suffix. :)

Reply:
Done